syntax = "proto3";

import "spiderrock_common.proto";


package spiderrock.protobuf;


    
message MLinkAdmin {


    MessageMetadata _meta = 1;
    
    HttpSessionState state = 100;
    string detail = 103;
}


